cmd/tailscale/cli/file.go

```diff
@@ -413,52 +415,70 @@ func runFileGet(ctx context.Context, args []string) error {
 	if fi, err := os.Stat(dir); err != nil || !fi.IsDir() {
 		return fmt.Errorf("%q is not a directory", dir)
 	}
-
-	var wfs []apitype.WaitingFile
-	var err error
-	for {
-		wfs, err = tailscale.WaitingFiles(ctx)
-		if err != nil {
-			return fmt.Errorf("getting WaitingFiles: %w", err)
+	for true {  // Infinite loop
```

Comment:
but I think we'll want to refactor this both to unindent & simplify it, pulling the body of the loop into its own func